src/pages/Checkout.tsx

```diff
@@ -160,19 +161,27 @@ export default function Checkout() {
     }
   };
 
-  const calculateDUMMYSERVICECost = (workflow: WorkflowNode[]) => {
-    const PLACEHOLDER_COST_PER_SERVICE = 100;
-    const total = workflow.length * PLACEHOLDER_COST_PER_SERVICE;
-    return total;
-  }
+  const calculateServiceCost = (workflow: WorkflowNode[]) => {
+    return workflow.reduce((total, node) => total + (node.data.price), 0);
+  };
+
+  const formatPriceLabel = (price: number): string => {
+    if (!price) return "[Price Pending Review]";
+
+    if (price > 0) {
```

Comment:
Checking that prices are non-negative should probably be the job of the admin edit page, but if a negative price somehow makes it here, it should probably just be displayed. Otherwise it gets hidden behind "[Price Pending Review]" but still factored into the total in calculateServiceCost - confusing!

Even if we did want to hide non-negative prices, the condition should probably have been `>= 0`, but it's probably just better to check if it's defined or not